go-controller/pkg/node/gateway_localnet.go

```diff
@@ -83,6 +83,10 @@ func newLocalGateway(nodeName string, hostSubnets []*net.IPNet, gwNextHops []net
 				if err := initExternalBridgeDropForwardingRules(exGwBridge.bridgeName); err != nil {
 					return fmt.Errorf("failed to add forwarding block rules for bridge %s: err %v", exGwBridge.bridgeName, err)
 				}
+			} else {
```

Comment:
could you please add a unit test for this; you can see samples: https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/node/gateway_localnet_linux_test.go

1. do one for addition with config=true
2. then use the same test to change config=false and then re-run to see if we are deleting the rules

Reply:
Ack